### ISSUE TITLE
Cannot import existing models

### ISSUE THREAD

**Issue by hugobuddel** [pahjbo/vodsl#13]
The instruction movie suggests to import the existing models into Eclipse: https://youtu.be/xzSk413raLY?t=536

However, this does not work; Eclips gives an "No projects are found to import" error:
![vodslnoprojectsfound](https://user-images.githubusercontent.com/1306453/58462864-08a72e00-8133-11e9-8c0b-bcb8a6a3afbd.png)

This is apparently because [there is no .project and .classpath](https://stackoverflow.com/questions/2638016/why-no-projects-found-to-import).

(The submodule is correctly cloned with `--recurse-submodules`, so the directory does contain the models.)

**Comment by hugobuddel** [comment]
"File" -> "Import" -> "File System" (just below "Existing Projects into Workspace") does seem work

**Comment by pahjbo** [comment]
I will check in a .project file in this case - it should be noted that the fact that the the models project has been made a git submodule of the main code was done as a sort of convenience, and it is better in most cases to clone the models project separately as the plug-in source sub-module reference is not absolutely guaranteed to reflect the current "latest" state of the models project.

**Comment by hugobuddel** [comment]
Yes, I initially did clone the repositories separately. This is a minor issue. You might also add a correction to the description of the video.
